src/pages/ReportParticipantsPage.tsx

```diff
@@ -63,15 +63,9 @@ function ReportParticipantsPage({report, personalDetails, session}: ReportPartic
     const isCurrentUserAdmin = ReportUtils.isGroupChatAdmin(report, currentUserAccountID);
     const getUsers = useCallback((): MemberOption[] => {
         let result: MemberOption[] = [];
-        ReportUtils.getVisibleChatMemberAccountIDs(report.reportID).forEach((accountID) => {
+        ReportUtils.getParticipantAccountIDs(report.reportID).forEach((accountID) => {
             const role = report.participants?.[accountID].role;
             const details = personalDetails?.[accountID];
-
-            if (!details) {
-                Log.hmmm(`[WorkspaceMembersPage] no personal details found for policy member with accountID: ${accountID}`);
-                return;
-            }
-
```

Comment:
Revert. This caused a crash where a member was hidden

Reply:
Ah hmm.. seems weird to not show them at all. I am fine with returning early for now. The admin/report creator should be able to see them.